### ISSUE TITLE
[BUG] Confusing screen reader experience re-ordering Favorites on the Customize Home page

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#5754]
**Describe the bug**
Confusing screen reader experience re-ordering Favorites on the Customize Home page. 
Assistive technology will announce the name of the favorite, then its position in the list, and how to swipe up or down to adjust where it is in position. When the user swipes up or down they can only move an item one position in either direction and then it is as if the focus is lost. Then, the double tap is required to move up and down. 

Since the gestures of a screen reader mimic the way a keyboard user navigates, this functionality lacking proper instructions (and/or not operating as it instructs) violates [WCAG 2.1.1 (Keyboard)](https://www.w3.org/TR/UNDERSTANDING-WCAG20/keyboard-operation-keyboard-operable), as users cannot move items, and[ 4.1.2 (Name, Role, Value)](https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-rsv.html), as the "grabbed" state is not announced.

**To Reproduce**
_Using TalkBack or VoiceOver_
1. Go to 'Customize.'
2. Gesture to 'Current Favorites'

**Expected behavior**
Assistive technology will announce the name of the favorite, then its position in the list, and how to swipe up or down to adjust where it is in position. When the user swipes up or down they can continue to do so until the item is in the position they want. 

_(my previous understanding was that the double tap was necessary before swiping up/down. Is this a bug that the user is allowed to move once? Or is it a bug that it is not allowing them to continue to? I'm not sure why it allows only one move in either direction at all without the double tap.)_

**Actual behavior**
Assistive technology will announce the name of the favorite, then its position in the list, and how to swipe up or down to adjust where it is in position. When the user swipes up or down they can only move an item one position in either direction and then it is as if the focus is lost. Then, the double tap is required to move up and down. 

 1 - should the users need to double tap to be able to swipe up/down?
2 - If they should, why are they allowed one move in either direction prior to double tapping?

**Screenshots**
Please see behavior on video, specifically "Visited Recently". It was 2nd from last in the Favorites section. This would mean to (partially sighted screen reader user) that user should have to swipe down 2 times to move it out of Favorites, and due to previous instruction, I will be told when removing. I swiped down 6 times before the message asking to Remove from Favorites was announced to screen reader.  I'm unclear if the amount of bugs here. 

https://github.com/user-attachments/assets/ef67ef1e-2f79-41ff-b84a-80f64df8ecad

**Smartphone (please complete the following information):**
- Device: Google Pixel 8a
 - OS:  Android 16
 - Version 7.3.51

 - Device: iPhone 14+
 - OS:  18.6.2
 - Version 7.3.51

**Additional context**
